drone_num:                  3.0

init_pos_0_x:               0.0
init_pos_0_y:               0.0
init_pos_0_z:               100.0

init_pos_1_x:               0.0
init_pos_1_y:               -300.0
init_pos_1_z:               100.0

init_pos_2_x:               0.0
init_pos_2_y:               300.0
init_pos_2_z:               100.0

init_vel_0_x:               150.0
init_vel_0_y:               0.0
init_vel_0_z:               0.0

init_vel_1_x:               150.0
init_vel_1_y:               0.0
init_vel_1_z:               0.0

init_vel_2_x:               150.0
init_vel_2_y:               0.0
init_vel_2_z:               0.0

des_theta_0_x:              1.0
des_theta_0_y:              0.0
des_theta_0_z:              0.0

des_theta_1_x:              0.707
des_theta_1_y:              -0.707
des_theta_1_z:              0.0

des_theta_2_x:              0.707
des_theta_2_y:              0.707
des_theta_2_z:              0.0
